AllJoyn/Platform/Bridge/BridgeNET/BridgeNET.vcxproj

```diff
@@ -211,7 +212,9 @@
     <ClInclude Include="Stdafx.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="AssemblyInfo.cpp" />
+    <ClCompile Include="AssemblyInfo.cpp">
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Release|x64'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
I'm specifically referring to how this one file and one configuration is set to use MTD runtime library. Was that necessary or just a mistake?

---
In reply to: [75537238](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75537238) [](ancestors = 75537238,75174478)